src/main/ionic/peers/src/pages/myprofile/myprofile.html

```diff
@@ -0,0 +1,66 @@
+<ion-header>
+  <ion-navbar>
+    <button ion-button menuToggle>
+      <ion-icon name="menu"></ion-icon>
+    </button>
+    <ion-title>My Profile</ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content>
+  <ion-list>
+    <button ion-item *ngFor="let item of items" (click)="itemTapped($event, item)">
+      <ion-icon [name]="item.icon" item-start></ion-icon>
+      {{item.title}}
+      <div class="item-note" item-end>
+        {{item.note}}
+      </div>
+    </button>
+  </ion-list>
+  <div *ngIf="selectedItem" padding>
+    You navigated here from <b>{{selectedItem.title}}</b>
+  </div>
+  <ion-grid *ngIf="user">
+    <ion-row>
+      <ion-col class="card-container">
+        <ion-row>
+          <ion-col text-center>
```

Comment:
Can you make the columns responsive so they won't be full width on a pc screen? You can use col-md offset-md:
https://ionicframework.com/docs/theming/responsive-grid/